LICENSES/LicenseRef-scancode-unicode.txt

```diff
@@ -0,0 +1 @@
+For terms of use and license, see https://www.unicode.org/terms_of_use.html
```

Comment:
maybe you can copy-paste the license from here:

https://scancode-licensedb.aboutcode.org/unicode.html